bedrock/firefox/templates/firefox/browsers/index.html

```diff
@@ -28,6 +28,8 @@
 {% set referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=firefox-browsers' %}
 {% set _entrypoint = 'mozilla.org-firefox-browsers' %}
 
+{% include '/includes/sticky-promo.html' %}
```

Comment:
Inserting the promo outside of `{% block content %}` here results in the `<aside>` coming before the `<head>` in the DOM. The effect is a visible flash of unstyled HTML before the CSS loads in.

We should also consider SEO here. This promo probably isn't the most important content in the page, so can probably come last at the bottom of the page?

Reply:
Moved.